### ISSUE TITLE
Dual Mining, please help me i cant figure it out.

### ISSUE THREAD

**Issue by imrecsoka** [nicehash/NiceHashMiner-Archived#963]
I want to know how to DUAL MINE with Claymore EthDcrMiner but i dont know what the hell the ExtraLaunchParameters are

**Comment by ttupper92618** [comment]
It's pretty easy actually.

- Look in the miner directory for the configs folder
- open the benchmark files for each GPU you want to dual-mine with
- Scroll all the way down to where you see "Name": "ClaymoreAMD_DaggerHashimoto",
- In the "ExtraLaunchParameters" section, add "-dual=Decred", as in "ExtraLaunchParameters": "-dual=Decred",
- Save the files and restart niceHashMiner

I would not recommend you do this with older AMD cards.  It may be fine on a 580, but it makes a very pronounced hashrate difference on older cards( 470 / 480).  Really you need to look at profitability and crunch the numbers.  As it stands, dual mining with those older cards is less profitable than ETH only, in my opinion.

**Comment by imrecsoka** [comment]
Hi thank you very much you at least explain properly.
Now another frustration how the hell do i change the coin.
I have put in -Dual=Siacoin and it doesnt bloody work, this is so bloody ridiculous really.
I cant find bloody documentation on what the strings should be
mining is a bloody headache but i need to try this because thingsn are so darn expensive and i cant afford my junk every month wife doesnt work its bloody hard

**Comment by ttupper92618** [comment]
I don't know if NiceHash will support Siacoin.  I am only aware of these three options:

-dual=Decred
-dual=Lbry
-dual=Pascal

**Comment by ttupper92618** [comment]
IMO if you are GPU mining, it would be best to do ETH at the present time.

**Comment by imrecsoka** [comment]
yeh but i want to dual mine eth and sia, I know it is possible:   SC is Siacoin , but bloody hell i dont know how to get it to work.... 
![image](https://cloud.githubusercontent.com/assets/25494601/26696058/69b5d190-470d-11e7-83ac-6cbb312b017b.png)
I found the documenation on the site some how and i see it shows what u are showing me BUT i bet like almost all sites the documentation is not up to date.
I am convinced u can ETH and SIA mine, 

Anyways , i tried to use:
-dual=Decred    = works
-dual=Pascal      = doesnt bloody work
not going to bther with lbry

what the heck is going on why is this so frustrating to get right

**Comment by ttupper92618** [comment]
It would probably be better for you to sign up with some pools and use Claymore directly instead of via NiceHash.  

http://miningclub.info/resources/claymores-dual-ethereum-decred-siacoin-lbry.1/

In any case, this isn't the right place for this - this is for reporting bugs related to NiceHashMiner.  As this isn't, so far as I can see, a supported feature it would be better to issue a feature request.

**Comment by imrecsoka** [comment]
correct, thanks for your patience and i will have a good read  of the site u sent me, thank you again.
I am just trying to determine  which automatic mining system is the best.
I so far know of two, one of which is NiceHashMiner

**Comment by ShrinivasKR** [comment]
For now, you can run Claymore directly, connecting to Nicehash pools for dual mining Eth and Sia.

**Comment by imrecsoka** [comment]
ShrinivasKR  sounds great, can you show me how please?
